Added functionality to switch to an existing R terminal

gowerc opened the pull request:
**What problem did you solve?**

#337  - Though feel free to reject if you don't like the proposal :) 

**(If you have)Screenshot**

![2020-05-23 21 51 40](https://user-images.githubusercontent.com/8447209/82740371-155b6380-9d40-11ea-9a80-cf8a66c4856d.gif)

renkun-ken commented on src/rTerminal.ts:
```diff
@@ -71,6 +71,13 @@ export async function chooseTerminal(active: boolean = false) {
             if (rTermNameOpinions.includes(activeTerminalName)) {
```
Is it `rTermNameOpinions` or `rTermNameOptions`? Looks like a typo introduced a while ago? Or I'm missing something?

andycraig replied:
Definitely a typo. (Possibly by me.)

gowerc commented:
@andycraig  & @renkun-ken 

Added a commit to fix the typo you mentioned.
Also added a commit to try to select the most "recent" R terminal. (more details in #337 )

renkun-ken commented:
What happens if there are multiple open terminals but none is named as `['R', 'R Interactive']`? It would be nice to show a message or create a new terminal in this case?

gowerc commented:
@renkun-ken  -  If there aren't any terminals called R then rTerm is left undefined `undefined` and gets picked up by the block [here](https://github.com/Ikuyadeu/vscode-R/blob/ffeab34ec5bf7fe7131fb75aae6d18ce44ebf087/src/rTerminal.ts#L98) and thus attempts to create a new terminal (printing an error message if its unable to do so).

renkun-ken commented:
Thanks, just forgot the case has been handled previously.

LGTM.
